addons/payment_mercado_pago/models/payment_transaction.py

```diff
@@ -58,6 +58,19 @@ def _mercado_pago_prepare_preference_request_payload(self):
         webhook_url = urls.url_join(
             base_url, f'{MercadoPagoController._webhook_url}/{self.reference}'
         )  # Append the reference to identify the transaction from the webhook notification data.
+
+        # In the case we are issuing a preference request in Chilean Pesos we must ensure that
+        # the price unit is an integer as Chilean Pesos do not have an small unit.
+        unit_price = self.amount
+        if self.currency_id.name in ('CLP', 'COP'):
+            error_msg = {'CLP': 'Chilean Pesos', 'COP': 'Colombian Pesos'}
+            rounded_unit_price = int(self.amount)
+            if rounded_unit_price != self.amount:
+                raise UserError(
+                    _("For {} prices must be integers."), error_msg[self.currency_id.name]
```

Comment:
```suggestion
                    _("For %s prices must be integers.", error_msg[self.currency_id.name])
```

Reply:
The message itself is not super clear IMHO.

What about "Prices in the currency CLP|COP must be expressed in integer values."? Don't bother interpolating the currency name if you don't translate it.